tests/kafkatest/services/kafka/kafka.py

```diff
@@ -449,6 +451,25 @@ def create_topic(self, topic_cfg, node=None):
         for line in self.describe_topic(topic_cfg["topic"]).split("\n"):
             self.logger.info(line)
 
+    def delete_topic(self, topic, node=None):
+        """
+        Delete a topic with the topics command
+        :param topic:
+        :param node:
+        :return:
+        """
+        if node is None:
+            node = self.nodes[0]
+        self.logger.info("Creating topic %s with settings %s",
```

Comment:
why is delete topic method creating a topic? The code below also has `--create` option. Maybe I am missing something.